region/norfair/east/Acid Snakes Tunnel.json

```diff
@@ -186,6 +188,11 @@
     }
   ],
   "strats": [
+    {
+      "link": [1, 1],
+      "name": "Crystal Flash",
+      "requires": ["h_canHeatedCrystalFlash"]      
```

Comment:
True. I was thinking it could still be useful in some scenario, since after doing the CF you would take more heat damage running from the farm compared to doing CF at the door. But even then it's almost certainly preferable to conserve the ammo, e.g. to do a CF in the next room instead. I can delete it.